system-int-test/test/cmr/system_int_test/ingest/collection_direct_distribution_fields_test.clj

```diff
@@ -0,0 +1,65 @@
+(ns cmr.system-int-test.ingest.collection-direct-distribution-fields-test
```

Comment:
No need to add this test. We will do the testing when the search ticket is worked.